thrift/bitset.go

```diff
@@ -0,0 +1,70 @@
+package thrift
+
+import "errors"
+
+const size = 64
+
+type bits uint64
+
+// BitSet is a set of bits that can be set, cleared and queried.
+type BitSet []bits
```

Comment:
Shouldn't export this as it's not functionality that should be relied upon by users of the package.